src/controller/CommissionerDiscoveryController.h

```diff
@@ -38,6 +38,7 @@
 
 #if CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY
 
+using chip::CharSpan;
```

Comment:
Please don't put `using` at toplevel in headers.  I know the code here was already doing that for some types.  It shouldn't be.

Reply:
Yup, did it to align with the existing code, but would rather avoid it. As long as it's not a problem to be inconsistent with the current style, I will change it.

Reply:
top-level `using` should practically never be used in public headers. If there are other usages, they should be removed/replaced rather than copied.

Reply:
Thanks, good to know. I will remove all top-level `using` uses in this file to be consistent.

Reply:
Code removed, fixed.